CHANGELOG.next.asciidoc

```diff
@@ -316,6 +316,16 @@ automatic splitting at root level, if root level element is an array. {pull}3415
 - Add XML decoding support to HTTPJSON. {issue}34438[34438] {pull}35235[35235]
 - Add delegated account support when using Google ADC in `httpjson` input. {pull}35507[35507]
 - Allow specifying since when to read journald entries. {pull}35408[35408]
+- Add metrics for filestream input. {pull}35529[35529]
+- Add support for collecting `httpjson` metrics. {pull}35392[35392]
+- Add XML decoding support to CEL. {issue}34438[34438] {pull}35372[35372]
+- Mark CEL input as GA. {pull}35559[35559]
+- Add metrics for gcp-pubsub input. {pull}35614[35614]
+- [GCS] Added scheduler debug logs and improved the context passing mechanism by removing them from struct params and passing them as function arguments. {pull}35674[35674]
+- Allow non-AWS endpoints for awss3 input. {issue}35496[35496] {pull}35520[35520]
+- Add Okta input package for entity analytics. {pull}35611[35611]
+- Expose harvester metrics from filestream input {pull}35835[35835] {issue}33771[33771]
+- Add device support for Azure AD entity analytics. {pull}35807[35807]
```

Comment:
It seems some extra entries came when updating the PR and the one from this PR is missing.